ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/BeaconState.java

```diff
@@ -172,46 +170,51 @@ public BeaconState get_initial_beacon_state(
     ArrayList<CrosslinkRecord> latest_crosslinks = new ArrayList<>(SHARD_COUNT);
 
     for (int i = 0; i < SHARD_COUNT; i++) {
-      latest_crosslinks.add(
-          new CrosslinkRecord(Bytes32.ZERO, UnsignedLong.valueOf(INITIAL_SLOT_NUMBER)));
+      latest_crosslinks.add(new CrosslinkRecord(Bytes32.ZERO, GENESIS_SLOT));
     }
 
-    BeaconState state =
-        new BeaconState(
-            // Misc
-            INITIAL_SLOT_NUMBER,
-            genesis_time,
-            new Fork(
-                UnsignedLong.valueOf(INITIAL_FORK_VERSION),
-                UnsignedLong.valueOf(INITIAL_FORK_VERSION),
-                UnsignedLong.valueOf(INITIAL_SLOT_NUMBER)),
-
-            // Validator registry
-            new Validators(),
-            new ArrayList<>(),
-            INITIAL_SLOT_NUMBER,
-            Bytes32.ZERO,
-
-            // Randomness and committees
-            latest_randao_mixes,
-            new ArrayList<>(),
-
-            // Finality
-            INITIAL_SLOT_NUMBER,
-            INITIAL_SLOT_NUMBER,
-            0,
-            INITIAL_SLOT_NUMBER,
-
-            // Recent state
-            latest_crosslinks,
-            latest_block_roots,
-            new ArrayList<>(),
-            new ArrayList<>(),
-            new ArrayList<>(),
-
-            // Ethereum 1.0 chain data
-            latest_eth1_data,
-            new ArrayList<>());
+    // todo after update v0.01 constants no longer exist
+    BeaconState state = new BeaconState();
+    //    BeaconState state =
```

Comment:
Why is this commented out, rather than removed?

Reply:
Too many cascading changes when modifying the BeaconState constructor.